lighthouse-cli/test/smokehouse/test-definitions/dobetterweb/dbw-expectations.js

```diff
@@ -440,6 +440,33 @@ const expectations = [
   //     },
   //   },
   // },
+  {
+    artifacts: {
```

Comment:
`blockedByResponse` issue type won't land until M85 https://chromiumdash.appspot.com/commit/f2ea618e6ba4d520b78ce97d20b53ff78de5836e so I guess we gotta comment this out too.